components/calendar/src/types.rs

```diff
@@ -108,9 +108,9 @@ impl FormattableYear {
 pub struct MonthCode(pub TinyStr4);
 
 impl MonthCode {
-    /// Returns an option which is Some containing the non-month version of a leap month
-    /// if the MonthCode this method is called upon is a leap month, and None otherwise.
-    /// This method assumes the MonthCode is valid.
+    /// Returns an option which is `Some` containing the non-month version of a leap month
+    /// if the `MonthCode` this method is called upon is a leap month, and `None` otherwise.
+    /// This method assumes the `MonthCode` is valid.
```

Comment:
Suggestion: where able, we should try to make these links. Even reflexive links are useful because they get tracked with renames and other refactors.

```suggestion
    /// if the [`MonthCode`] this method is called upon is a leap month, and `None` otherwise.
    /// This method assumes the [`MonthCode`] is valid.
```

Reply:
Okay, will improve upon this as well.